src/pages/GetStarted/GetStartedResults.vue

```diff
@@ -118,14 +152,113 @@ export default {
 	metaInfo: {
 		title: 'Results - Get Started'
 	},
+	computed: {
+		countVerb() {
+			return this.loanIds.length === 1 ? 'is' : 'are';
+		},
+		countNumber() {
+			switch (this.loanIds.length) {
+				case 1:
+					return 'one';
+				case 2:
+					return 'two';
+				default:
+					return 'three';
+			}
+		},
+		countPeople() {
+			return this.loanIds.length === 1 ? 'person' : 'people';
+		},
+		countNeed() {
+			return this.loanIds.length === 1 ? 'needs' : 'need';
+		},
+	},
+	apollo: {
+		preFetch: true,
+		preFetchVariables() {
+			return {
+				limit: 3,
+				visitorId: cookieStore.get('uiv') || null
+			};
+		},
+		variables() {
+			return {
+				limit: 3,
+				visitorId: cookieStore.get('uiv') || null
+			};
+		},
+		query: gql`query getStartedResults($limit: Int, $visitorId: String) {
+			general {
+				lendingPreferences(visitorId: $visitorId) {
+					loans(limit: $limit) {
+						totalCount
+						values {
+							id
+						}
+					}
+				}
+			}
+			# TODO: might do this in a follow-up query instead of in the initial query
+			lend {
+				loans(limit: $limit) {
+					values {
+						id
+					}
+				}
+			}
+		}`,
+		result(result) {
+			// TODO: error handling
+			let loanValues = result.data?.general?.lendingPreferences?.loans?.values || [];
```

Comment:
Have we confirmed that our babel setup transpiles this correctly for IE11?

Reply:
Not yet, but I'll check that before moving this out of draft

Reply:
Confirmed, babel is transpiling this! This particular line is transformed into this:
```
var g=(null===(e=t.data)||void 0===e||null===(n=e.general)||void 0===n||null===(s=n.lendingPreferences)||void 0===s||null===(a=s.loans)||void 0===a?void 0:a.values)||[];
```

Reply:
💯